integration_tests/2pc_test.go

```diff
@@ -1742,3 +1743,90 @@ func (s *testCommitterSuite) TestFlagsInMemBufferMutations() {
 		s.Equal(assertNotExist, mutations.IsAssertNotExist(i))
 	})
 }
+
+func (s *testCommitterSuite) TestSetLockedKeyValue() {
+	ctx := context.Background()
+	k1 := []byte("k1")
+	v1 := []byte("v1")
+	v2 := []byte("v2")
+
+	mustLockKey := func(txn transaction.TxnProbe, key []byte) {
+		s.Require().NoError(txn.LockKeys(ctx, &kv.LockCtx{ForUpdateTS: txn.StartTS(), WaitStartTime: time.Now()}, key))
+	}
+	mutOpVals := func(opVals ...interface{}) func(m transaction.CommitterMutations) {
```

Comment:
I just have a question: what does `mut` mean here?

Reply:
It's the abbr of 'mutation'. The function means to create a check function from mutation op-val pairs. I just want to make the name shorter, but yes, it's little hard to understand.